packages/backend/src/utils/inferenceUtils.ts

```diff
@@ -115,6 +115,11 @@ export function generateContainerCreateOptions(
     throw new Error('The model info file provided is undefined');
   }
 
+  const envs: string[] = [`MODEL_PATH=/models/${modelInfo.file.file}`, 'HOST=0.0.0.0', 'PORT=8000'];
```

Comment:
missing a unit test to see if env variable contains CHAT_FORMAT if modelInfo has or not the chatformat attribute

Reply:
Forgot to push the commit 😪